lib/xmldoc.js

```diff
@@ -135,7 +135,7 @@ XmlElement.prototype.toStringWithIndent = function(indent, options) {
     if (Object.prototype.hasOwnProperty.call(this.attr, name))
         s += " " + name + '="' + this.attr[name] + '"';
 
-  var finalVal = this.val.trim().replace(/</g, "&lt;").replace(/>/g, "&gt;");
+  var finalVal = this.val.trim().replace(/</g, "&lt;").replace(/>/g, "&gt;").replace(/&/g, '&amp;');
 
   if (options && options.trimmed && finalVal.length > 25)
```

Comment:
Done.  Actually, I only ran into issues with the ampersand, but saw this StackOverflow post and tried to be complete: http://stackoverflow.com/questions/1091945/what-characters-do-i-need-to-escape-in-xml-documents.   I see now reading the detailed comments that you're right, double and single quotes need not be escaped.  Thanks!